dspy/dsp/utils/settings.py

```diff
@@ -7,6 +7,7 @@
 DEFAULT_CONFIG = dotdict(
     lm=None,
     adapter=None,
+    backup_adapter=None,
```

Comment:
is this intentional? I remember this is in another pending PR 🤔

Reply:
This shouldn't be necessary, I guess Isaac added this but didn't use it in the two step adapter implementation.